superset-frontend/src/utils/pathUtils.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { applicationRoot } from 'src/utils/getBootstrapData';
+
+/**
+ * Takes a string path to a resource and prefixes it with the application root that is
+ * defined in the application configuration. The path is unsanitized.
+ * @param path A string path to a resource
+ */
+export function ensureAppRootUnsanitized(path: string): string {
```

Comment:
I'm not sure if I understand what a Sanitized or Unsanitized path means in this context (?)

Reply:
I'm not wild on the names but was trying to emphasize that this does nothing to the app root itself before prefixing it (see the next comment below regarding a security warning I was trying to address).